packages/inference/src/HfInference.ts

```diff
@@ -697,6 +701,12 @@ export class HfInference {
 		if (options?.binary && mergedOptions.wait_for_model) {
 			headers["X-Wait-For-Model"] = "true";
 		}
+		if (options?.binary && mergedOptions.use_cache === false) {
+			headers["X-Use-Cache"] = "false";
+		}
+		if (options?.binary && mergedOptions.dont_load_model) {
+			headers["X-Load-Model"] = "0";
+		}
```

Comment:
Maybe those could be grouped inside a `if (options?.binary) {`

Reply:
handled in be4ee36e3a97c0744efd00887b2390caf2de3140